README.md

```diff
@@ -145,6 +145,28 @@ In the above case only the `presence` validator will be passed to the client.
 
 This is also the case with [other supported conditional validations](http://guides.rubyonrails.org/v4.2.0/active_record_validations.html#conditional-validation) (such as Procs, Arrays or Strings).
 
+**NOTE:** when `:if` conditional includes a symbol or a string with
+`changed?` in it, validator will always run. The conditional itself will
+not be run at all in this case.
+
+```ruby
+class Person < ActiveRecord::Base
+  validates :name, presence: true, if: :name_changed?
+
+  def name_changed?
+    # this never runs, but presence of name is always validated
+  end
+end
+```
+
+In the above code the presence of name is always validated, but the
+`name_changed?` method is never run.
+
+This is done because it is always assumed the value will change on the
+form.
+
+Conditionals defined with `:unless` key do not have this optimization.
```

Comment:
It is correct, but I do not expect people to override `attribute_changed?` methods. I think it is referring to the default active model methods http://api.rubyonrails.org/classes/ActiveModel/Dirty.html 

I don't know if this could be confusing